modopt/opt/proximity.py

```diff
@@ -546,3 +547,138 @@ def _cost_method(self, *args, **kwargs):
             print(' - OWL NORM (X):', cost_val)
 
         return cost_val
+
+
+class Ridge(ProximityParent):
+    """L2-norm proximity operator (i.e. shrinkage)
+
+    This class defines the L2-norm proximity operator
+    prox(y) = argmin 0.5||x-y||_2^2 + alpha*||x||_2^2
+              x in C
```

Comment:
Not necessary, but if you wanted you could have this equation render in LaTeX for the API docs. See *e.g.* [this example](https://cea-cosmic.github.io/ModOpt/modopt.math.matrix.html#modopt.math.matrix.nuclear_norm).

Reply:
Replaced